scripts/type_x509_req.py

```diff
@@ -60,8 +60,8 @@
   -- since there seems no way to increase ref count for a GENERAL_NAME
   -- we left the elements referenced by the new-dup'ed stack
   local got_ref = got
-  ffi_gc(got_ref, stack_lib.gc_of("GENERAL_NAME"))
-  got = ffi_cast("GENERAL_NAMES*", got_ref)''',
+  got = ffi_cast("GENERAL_NAMES*", got_ref)
+  ffi_gc(got, stack_lib.gc_of("GENERAL_NAME"))''',
```

Comment:
If you have set ffi.gc function for got_ref in the upper layer function,  then the underlying object would be freeed two times by the ffi.gc.

Reply:
It won't be double free because `got_ref` is a opaque pointer, it doesn't have GC handler to free the pointed object.